dataeng/resources/setup-exporter-email-optin.sh

```diff
@@ -16,8 +16,11 @@ pip install -r github_requirements.txt
 pip install mysql-connector-python -e .
 popd
 
+aws secretsmanager get-secret-value --secret-id analytics-secure/analytics-exporter/task-auth.json --region "us-east-1" --output json | jq -r ".SecretString" >>  {WORKSPACE}/dataeng/analytics-exporter/task-auth.json
+
+
 # Configuration paths in analytics-secure
-SECURE_ROOT=${WORKSPACE}/analytics-secure/analytics-exporter
+SECURE_ROOT=${WORKSPACE}/dataeng/analytics-exporter
```

Comment:
I believe this would not work as this repo(jenkins-job-dsl), does not get cloned as part of this job.